lib/cErrors.ml

```diff
@@ -124,9 +124,15 @@ let print_gen ~anomaly (e, info) =
     print_backtrace info ++ fnl() ++
     str "<original exception>:" ++ spc() ++ extra_msg ++ print_anomaly false e
 
+let print_quickfix = function
+  | [] -> mt ()
+  | qf -> fnl () ++ Quickfix.print qf
+
 (** The standard exception printer *)
 let iprint (e, info) =
-  print_gen ~anomaly:true (e,info) ++ print_backtrace info
+  print_gen ~anomaly:true (e,info) ++
+  Result.fold ~ok:print_quickfix ~error:(print_gen ~anomaly:true) (Quickfix.get e) ++
```

Comment:
shouldn't this be sent to feedback separately now?

Reply:
Hum, I think the error message business is still messy. In particular all errors are turned into user erorrs:
```
let process_expr ~state loc_ast =
  try interp_vernac ~check:true ~state loc_ast
  with reraise ->
    let reraise, info = Exninfo.capture reraise in

    (* Exceptions don't carry enough state to print themselves (typically missing the nametab)
       so we need to print before resetting to an older state. See eg #16745 *)
    let reraise = UserError (CErrors.iprint (reraise, info)) in
    (* Keep just the loc in the info as it's printed separately *)
    let info = Option.cata (Loc.add_loc Exninfo.null) Exninfo.null (Loc.get_loc info) in

    ignore(Stm.edit_at ~doc:state.doc state.sid);
    Exninfo.iraise (reraise, info)
```
If I feedback an error then it may be printed twice.

Should I pass to iprint a flag or have a twin API?
In general I think printing the quickfix as part of the error message may not be a bad idea, although what I print now is surely perfectible.

Reply:
Maybe, but if the reason is to make the code above sensible, I'm not so sure.

UserError is already used elsewhere, eg Ltac, while the code above should really throw a dedicated exception (with an explicit Pp, a loc and Quickfix). I'd rather leave this cleanup to another PR.